plugins/outputs/iotdb/README.md

```diff
@@ -0,0 +1,123 @@
+# IoTDB Output Plugin
+
+This output plugin saves Telegraf metrics to an Apache IoTDB backend,
+supporting session connection and data insertion.
+
+## Apache IoTDB
+
+Apache IoTDB (Database for Internet of Things) is an IoT native database with
+high performance for data management and analysis, deployable on the edge and
+the cloud. Due to its light-weight architecture, high performance and rich
+feature set together with its deep integration with Apache Hadoop, Spark and
+Flink, Apache IoTDB can meet the requirements of massive data storage,
+high-speed data ingestion and complex data analysis in the IoT industrial
+fields.
+
+For more details consult the [Apache IoTDB website](https://iotdb.apache.org)
+or the [Apache IoTDB GitHub page](https://github.com/apache/iotdb).
+
+## Getting started
+
+Before using this plugin, please configure the IP address, port number,
+user name, password and other information of the database server,
+as well as some data type conversion, time unit and other configurations.
+
+Please see the [configuration section](#Configuration) for an example
+configuration.
+
+## Please pay attention to these points
+
+1. IoTDB (version 0.13.x or older) **DO NOT support unsigned integer**.
+There are three available options of converting uint64, which are specified by
+parameter `uint64_conversion`.
+
+   - `int64_clip`, default option. If an unsigned integer is greater than
+   `math.MaxInt64`, save it as `int64`; else save `math.MaxInt64`
+   (9223372036854775807).
+   - `int64`, force converting an unsigned integer to a`int64`,no mater
+   what the value it is. This option may lead to exception if the value is
+   greater than `int64`.
+   - `text`force converting an unsigned integer to a string, no mater what the
+   value it is.
+
+2. IoTDB supports a variety of time precision, but no matter what precision,
+timestamp is stored in `Int64`, so users need to specify the unit of timestamp.
+Default unit is `nanosecond`.
+
```

Comment:
Maybe
```suggestion
2. IoTDB supports a variety of time precision. You can specify which precision you want using the `timestamp_precision` option. Default is `nanosecond`.

```

Reply:
Great! Non-programmers really don't need to care about these details.